### PR TITLE
Issue 24: Update dependencies

### PR DESCRIPTION
**Pull request by natseg**
Let me know what you think.

I have run tests and build locally as well as tried it via `npm link` in our repo.
It seems to work nicely with no apparent breakage.

I was not able to update stack-trace, it has not been updated in a while.
So maybe something to think about, I will create an issue and see if I have some time to find an alternative if that's ok with you?

### CODE REVIEW

**Comment by chris-heathwood-uoy** [comment]
References https://github.com/edorivai/nock-record/issues/24 👍

File: package.json [hunk 1]
```diff
@@ -35,6 +35,11 @@
       "prettier --parser typescript --write"
     ]
   },
+  "prettier": {
```
Comment (edorivai) [inline package.json]: Please remove the prettier config here, let's just roll with the default config.

File: examples/githubProfile.ts [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
+            .get(`https://api.github.com/users/${username}`)
+            .set('User-Agent', 'edorivai/nock-record example');
 
-	const { url, name, location } = response.body;
-	return { url, name, location };
-}
\ No newline at end of file
+        const { url, name, location } = response.body;
+        return { url, name, location };
+    } catch {}
```
Comment (edorivai) [inline examples/githubProfile.ts]: What's the reason for the try/catch here?

File: package.json [hunk 3]
```diff
@@ -56,7 +56,7 @@
     "typescript": "^4.0.2"
   },
   "peerDependencies": {
-    "nock": "^11.0.0"
+    "nock": "^13.0.4"
```
Comment (edorivai) [inline package.json]: I might be misunderstanding here, but why would we want to bump the peerDependency? Wouldn't `^11.0.0` allow people to use v13 if they want?

**Review by edorivai — CHANGES_REQUESTED** [review]

**Comment by edorivai** [comment]
@natseg Thanks so much for the thorough PR, I've posted some questions, nothing major I think.

File: examples/githubProfile.ts (hunk 2 above)
Comment (natseg) [reply examples/githubProfile.ts]: Github had made some breaking changes to their API.
Having a `try {} catch {}` is a bit more foolproof? I am happy to remove :)

File: package.json (hunk 3 above)
Comment (natseg) [reply package.json]: I have changed this to a range, to be explicit about supporting latest version range but keep the support for 11 and more. I hope it's ok. 😄

**Pull request merged**